Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqTable.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqTable" {
+
+    It "should list tables from a dataset object from pipeline."{
+        $tables = Get-GcbqDataset “pshell_testing” | Get-GcbqTable
```

Comment:
This sample data is not set up in the test. Does that take time to do? If not much more than querying the data, the test should set up and tear down the sample data itself (in a BeforeAll and an AfterAll).